modules/developers-guide/pages/tutorials/explore-templates.adoc

```diff
@@ -178,7 +178,7 @@ For now, you can continue to the next section.
 [[start-the-local-network]]
 == Start the local network
 
-Before you can build the default project, you need to connect to the {IC} network either running *locally* in your development environment or running *remotely* on a subnet that you can access.
+Before you can build the default project, you need to connect to the {IC} network either running *locally* in your development environment or running *remotely* (on-chain) on a subnet that you can access.
```

Comment:
This should be "local canister execution environment" vs. "Internet Computer/IC/...". There is no "IC network running locally".